README.md

```diff
@@ -124,10 +127,41 @@ Put the `PUSH_CREDENTIAL_SID` configuration info into your application server by
     PUSH_CREDENTIAL_SID = 'CR***'
     APP_SID = 'AP***'
 
-Once you’ve done that, restart the server so it uses the new configuration info. Now it's time to test. Hit your application server's `placeCall` endpoint. This will trigger a Twilio REST API request that will make an inbound call to your mobile app. Once your app accepts the call, you should hear a congratulatory message.
+Once you’ve done that, restart the server so it uses the new configuration info. Now it's time to test. Hit your application server's `placeCall` endpoint. This will trigger a Twilio REST API request that will make an inbound call to your mobile app. You will receive a incoming call notification.
 
 <img height="500px" src="images/quickstart/incoming_notification.png">"
 
+if your applicaion is in the foreground, you will see an alert dialog.
+
+<img height="500px" src="images/quickstart/incoming_call.png">"
+
+Once your app accepts the call, you should hear a congratulatory message.
+
+### <a name="bullet10"></a>10. Make client to client call
+
+To make client to client calls, follow step 6 through step 8 to run the application in two devices. Press the call button to connect to Twilio.  
+
+<img height="500px" src="images/quickstart/voice_make_call_dialog.png">
+
+Enter a client name and press the call button to place a call.
+
+<img height="500px" src="images/quickstart/make_call_to_client.png">
+
+### <a name="bullet11"></a>11. Make client to PSTN call
+
+A verified phone number is one that you can use as your Caller ID when making outbound calls with Twilio. This number has not been ported into Twilio and you do not pay Twilio for this phone number.
+
+To make client to number calls, first get a valid Twilio number to your account via https://www.twilio.com/console/phone-numbers/verified. Update `server.py` and replace `CALLER_NUMBER` with the verified number.  
+
+Follow step 6 through step 8 to run the application in a device. Press the call button to connect to Twilio.
```

Comment:
Do they need to do 6 to 8 to make an outgoing call to a number? This looks like its specific to receiving incoming notifications.

Reply:
running step 6 should be sufficient in this case. In one of the example, we should demo how to receive a call from a PSTN number.